doc/data/messages/d/docstring-first-line-empty/pylintrc

```diff
@@ -0,0 +1,2 @@
+[DESIGN]
+docstring-first-line-empty:yes
```

Comment:
```suggestion
docstring-first-line-empty=yes
```

Reply:
So why is this still failing? It literally has the first line empty in the docstring